python/trifinger_simulation/visual_objects.py

```diff
@@ -122,5 +123,65 @@ def set_state(self, position, orientation):
             self.body_id,
             position,
             orientation,
-            **self._kwargs,
+            physicsClientId=self._pybullet_client_id,
+        )
+
+
+class CuboidMarker(ObjectMarker):
+    """Visualize a Cuboid."""
+
+    def __init__(
+        self,
+        size,
```

Comment:
I think it would be less confusing if we have either `width` or `half_width` throughout as else we'd need to pass the half-width to create a collision object, and the full width to create a visual object. So, maybe we could pass a `half_width` here as well (same name to avoid confusion)